tests/integration_tests/functional/test_snapshot_version.py

```diff
@@ -186,3 +153,48 @@ def test_create_invalid_version(bin_cloner_path):
             str(error)
     else:
         assert False, "Negative test failed"
+
+
+def test_create_with_newer_virtio_features(bin_cloner_path):
+    """
+    Attempt to create a snapshot with newer virtio features.
+
+    @type: functional
+    """
+    builder = MicrovmBuilder(bin_cloner_path)
+    test_microvm = builder.build_vm_nano().vm
+    test_microvm.start()
+
+    # Init a ssh connection in order to wait for the VM to boot. This way
+    # we can be sure that the block device was activated.
+    iface = NetIfaceConfig()
+    test_microvm.ssh_config['hostname'] = iface.guest_ip
+    _ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
+
+    # Create directory and files for saving snapshot state and memory.
+    snapshot_builder = SnapshotBuilder(test_microvm)
+    _snapshot_dir = snapshot_builder.create_snapshot_dir()
+
+    # Pause microVM for snapshot.
+    response = test_microvm.vm.patch(state='Paused')
+    assert test_microvm.api_session.is_status_no_content(response.status_code)
+
+    # We try to create a snapshot to a target version < 0.26.0.
+    # This should fail because Fc versions < 0.26.0 don't support
+    # virtio notification suppression.
+    response = test_microvm.snapshot.create(
```

Comment:
We could test v0.23 - v0.25 here to have full coverage.

Reply:
Done.